src/officehours_ui/static/css/main.css

```diff
@@ -181,6 +181,10 @@ a:hover {
 	color: #0065a3;
 }
 
+.btn-warning:focus {
+	box-shadow: 0 0 0 0.2rem rgba(12, 61, 222, 0.5);
```

Comment:
Can you use a hex value here? It may make our lives easier in the future as we try to control all the different colors in the palette.

Reply:
The blue focus indicator looks okay to me, but if you wanted to stay closer to yellow, you could use a brown? I wasn't privy to all the discussion about what color the button should be to begin with. A blue button seems like it would be okay to me, but maybe it's not noticeable enough.

Reply:
I think RGBA is much easier to visualize at a glance and tweak, but I appreciate that the rest of the file is hex and it's better to be consistent. I encoded the 50% alpha as an extra hex digit at the end. 343f483

@teshsopher Any input on the color?